motoko/exchange_rate/dfx.json

```diff
@@ -0,0 +1,29 @@
+{
+  "canisters": {
+    "exchange_rate": {
+      "package": "exchange_rate",
+      "type": "motoko",
+      "main": "src/Main.mo",
+      "wasm": "target/wasm32-unknown-unknown/release/exchange_rate.wasm"
```

Comment:
This looks like something from a Rust project, not Motoko. Is this right?

Reply:
Right, I copied the `dfx.json` from rust folder to motoko.  Will remove this line.

Reply:
This is cleaned up.